### ISSUE TITLE
using ColorVectorSpace asserts

### ISSUE THREAD

**Issue by KristofferC** [JuliaLang/julia#29136]
`ColorVectorSpace v0.6.2`

on debug build:

```
julia> using ColorVectorSpace
[ Info: Precompiling ColorVectorSpace [c3611d14-8923-5661-9e6a-0046d554d3a4]
Assertion failed: (!jl_subtype((jl_value_t*)sig, (jl_value_t*)type)), function check_ambiguous_visitor, file /Users/kristoffer/julia/src/gf.c, line 1189.

signal (6): Abort trap: 6
in expression starting at /Users/kristoffer/.julia/packages/ColorVectorSpace/HeIVH/src/ColorVectorSpace.jl:292
__pthread_kill at /usr/lib/system/libsystem_kernel.dylib (unknown line)
Allocations: 2233455 (Pool: 2233166; Big: 289); GC: 3
```

**Comment by JeffBezanson** [comment]
I believe it's not a dup of the original #22592 concerning JLD2, but it is a dup of the later comment that got the same assertion failure using ColorVectorSpace. I'll close that issue first since I believe it was fixed a while ago, and my new PR fixes this one.

**Comment by JeffBezanson** [comment]
Fixed by #29139
